R/enron_download_data.R

```diff
@@ -1,4 +1,21 @@
+#' Downloads the Enron dataset from the 'StreamlineDataScience/enron-example'
+#' GitHub repository.
+#'
+#' @param path The destination file path for the downloaded data. If missing, a temporary location is used.
+#'
+#' @return Returns the file path to the downloaded dataset (invisibly).
 #' @export
-enron_download_file = function() {
-
+#'
+#' @examples
+#' enron_download_file()
+enron_download_file <- function(path) {
+  if (missing(path)) {
+    path <- tempfile(fileext = ".xls")
+  }
+  utils::download.file(
+    "https://raw.githubusercontent.com/StreamlineDataScience/enron-example/main/andrea_ring_000_1_1.pst.0.xls",
+    destfile = path,
+    method = "curl"
```

Comment:
You may want to change `mode = "wb"` to accommodate Windows, though in our case it's irrelevant, since our VMs are all linux.  See https://github.com/HenrikBengtsson/Wishlist-for-R/issues/30#issuecomment-386437898